pkg/controller/datavolume-controller.go

```diff
@@ -439,6 +447,20 @@ func (r *DatavolumeReconciler) getSnapshotClassForSmartClone(dataVolume *cdiv1.D
 	return "", errors.New("could not match snapshotter with storage class, falling back to host assisted clone")
 }
 
+func preferHostAssistedCopy(c client.Client) (bool, error) {
```

Comment:
I think we should do it a little more future proof. We are expecting to add more options in the future, so it would make sense to have a getPreferredCloneStrategy function that returns the value. Also this is not a generic function used by all controllers, so you can make part of the reconciler so you don't have to pass the client.
Something like
```go
func (r *DatavolumeReconciler) getPreferrredCloneStrategy() (CDICloneStrategy, error) {
...
}
```